### PR TITLE
adding support for bool in from_python

### PR DESCRIPTION
**Pull request by mmatera**
This would be a fix for issue #1014

### CODE REVIEW

File: mathics/core/expression.py [hunk 1]
```diff
@@ -89,6 +89,11 @@ def from_python(arg):
     number_type = get_type(arg)
     if arg is None:
         return Symbol('Null')
+    if isinstance(arg, bool):
```
Comment (rocky) [inline mathics/core/expression.py:92]: This is exactly what I was thinking too - thanks.

**Comment by GarkGarcia** [comment]
@mmatera @rocky I guess we're ready to merge this?

**Comment by rocky** [comment]
I think so - @mmatera ? I have in the back of my mind to add  a round-trip test for all types. But that can come later.

**Comment by mmatera** [comment]
I agree: it would be nice to add some examples, but we can do it later. Let's merge this as it is if it is OK for you

**Comment by rocky** [comment]
Fixed by #1020. The broader issue of handling `to_python()` remains a research project and probably not something to track in a issue without breaking this up into small pieces like this one (handle boolean) .

Of course adding more tests and round-trip tests is a worthwhile endeaver too, but that's also probably a separate issue.

**Pull request merged**